src/compas_model/interactions/interaction.py

```diff
@@ -14,12 +14,11 @@ class Interaction(Data):
     @property
     def __data__(self):
         # type: () -> dict
-        return {"name": self.name, "value": self.value}
+        return {"name": self.name}
 
-    def __init__(self, name=None, value=None):
-        # type: (str | None, object | None) -> None
+    def __init__(self, name=None):
```

Comment:
What would be the mechanism behind to carry information other than name and indices of the interface?

Would you need to implement your own class that inherits interface with attributes you need?

Reply:
One example I had:
1) I define interfaces through collision detection.
2) then add interfaces with a pair of "features"
3) lastly I add geometry from the interface to the individual element features.